src/db/bolt/network/network.go

```diff
@@ -0,0 +1,162 @@
+/*******************************************************************************
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *******************************************************************************/
+  package network
+
+import (
+	"encoding/json"
+
+	"common/errors"
+	bolt "db/bolt/wrapper"
+)
+
+const bucketName = "network"
+
+type Addr struct {
+	Wired map[string]string    `json:"wired"`
+	Wireless map[string]string `json:"wireless"`
+}
+
+type NetworkInfo struct {
+	Id       string `json:"id"`
+	IPv4 Addr       `json:"IPv4"`
+	RTT float64     `json:"RTT"`
+	// IPv4Wired map[string]string `json:"IPv4Wired"`
+	// IPv4Wireless map[string]string `json:"IPv4Wireless"`
```

Comment:
If this commented line is not used, please remove.

Reply:
okay